tests/scripts/analyze_outcomes.py

```diff
@@ -71,6 +71,27 @@ def _has_word_re(words: typing.Iterable[str],
         _has_word_re(_PSA_MECHANISMS_NOT_IMPLEMENTED)
 
     IGNORED_TESTS = {
+        'ssl-opt': [
+            # We don't run ssl-opt.sh with Valgrind on the CI because
+            # it's extremely slow. We don't intend to change this.
+            'DTLS client reconnect from same port: reconnect, nbio, valgrind',
+
+            # We don't have IPv6 in our CI environment.
+            # https://github.com/Mbed-TLS/mbedtls-test/issues/176
+            'DTLS cookie: enabled, IPv6',
+            # Disabled due to OpenSSL bug.
+            # https://github.com/openssl/openssl/issues/18887
+            'DTLS fragmenting: 3d, openssl client, DTLS 1.2',
+            # We don't run ssl-opt.sh with Valgrind on the CI because
+            # it's extremely slow. We don't intend to change this.
+            'DTLS fragmenting: proxy MTU: auto-reduction (with valgrind)',
```

Comment:
Minor: Maybe it would be practical to group the ssl-opt.sh/valgrind exceptions together.

Reply:
With strings, I kept the list in lexicographic order. Between a predictable order and grouping related exceptions, I don't think there's a clear winner.